docs/setting-up/client/google.md

```diff
@@ -0,0 +1,149 @@
+# Google Cloud Platform
+
+PMM can monitor MySQL or PostgreSQL instances hosted on the [Google Cloud Platform][GOOGLE_CLOUD].
+
+The connection can be direct, or indirect using [Cloud SQL Proxy][GOOGLE_CLOUD_SQL_PROXY].
+
+## MySQL
+
+1. [Set up a MySQL instance on Google Cloud][GOOGLE_CLOUD_MYSQL]
+
+2. The database server must be accessible by PMM Client. If PMM Client is not also hosted on GCP, you will need to add a network interface with a public interface.
```

Comment:
do we have documentation about it already, or maybe link to external doc about how to add it ?

Reply:
We decided not to explain how to do that, as it's specific to GCP and not PMM. With these cloud page (AWS, Azure, GCP) we have to assume the reader knows how to manage their own service.